tests/foreman/ui/test_computeresource_rhev.py

```diff
@@ -1131,3 +1135,168 @@ def test_positive_provision_rhev_with_host_group(self):
+                orgs=[self.org_name],
+                org_select=True,
+                locations=[self.loc_name],
+                loc_select=True
+            )
+            parameter_list_img = [
+                ['Name', image_name],
+                ['Operatingsystem', os.title],
+                ['Architecture', arch.name],
+                ['Username', self.rhev_img_user],
+                ['Password', self.rhev_img_pass],
+                ['uuid', self.rhev_img_name],
+            ]
+            self.compute_resource.add_image(cr_name, parameter_list_img)
+            self.assertIn(
+                image_name,
+                self.compute_resource.list_images(cr_name)
+            )
+            self.compute_resource.set_profile_values(
+                cr_name, COMPUTE_PROFILE_SMALL,
+                cluster=self.rhev_datacenter,
+                cores=1,
+                memory=1024,
+                network_interfaces=[
+                    dict(
+                        name='nic1',
+                        network=settings.vlan_networking.bridge,
+                    ),
+                ],
+                storage=[
+                    dict(
+                        size='10',
+                        storage_domain=self.rhev_storage_domain,
+                        bootable=True,
+                        preallocate_disk=True,
+                    ),
+                ]
+            )
+            host_parameters_list = [
+                    ['Host', 'Organization', self.org_name],
+                    ['Host', 'Location', self.loc_name],
+                    ['Host', 'Host group', host_group.name],
+                    ['Host', 'Deploy on', cr_resource],
+                    ['Host', 'Compute profile', COMPUTE_PROFILE_SMALL],
+                    ['Operating System', 'Operating System', os.title],
+                    ['Operating System', 'Root password', root_pwd],
+                ]
+            make_host(
+                session,
+                name=host_name,
+                org=self.org_name,
+                loc=self.loc_name,
+                parameters_list=host_parameters_list,
+                provisioning_method='image',
+            )
+            vm_host_name = '{0}.{1}'.format(
+                host_name.lower(), domain.name)
+            # the provisioning take some time to finish, when done will be
+            # redirected to the created host
+            # wait until redirected to host page
+            self.assertIsNotNone(
+                session.hosts.wait_until_element(
+                    locators["host.host_page_title"] % vm_host_name,
+                    timeout=60
+                )
+            )
+            # Query RHEV hypervisor for vm description
+            vm_host_data = entities.Host().search_json(
+                query=dict(search='name="{0}"'.format(vm_host_name))
+            )['results'][0]
+            # Using documented RHEV API access
+            response = client.get(
```

Comment:
Any alternatives to `client.get` here? Such approach is strongly discouraged

Reply:
there is an expensive alternative, is to install a third party library RHEV sdk, also there is some libraries that do the same, but do not see the plus to do so for a one line command.
do not agree with you that this is discouraged as we are using a documented RHEV API call, using a standard http client by the way the same client used by nailgun, please let me know what is discouraged here?

Reply:
Note client.get  == requests.get , if you want I can invoke requests.get directly, but I do so I have to add it to requirements.

Reply:
nope, that's fine. I meant client.get is worse solution in comparison with `nailgun.entity.read()`
Check that out:
https://github.com/SatelliteQE/robottelo/issues/2837
So you are adding the code that should be refactored in future in advance. That is not good idea, but if no way and you sure on 100% that you cannot use nailgun for you purpose, so I will be fine with `client.get`

Reply:
It's strongly discouraged in most cases for Satellite API (you should use nailgun instead), but that's not the case here. This call is ok unless there's our framework to work with rhev api which i'm not aware of 🙂